cli/azd/pkg/account/manager.go

```diff
@@ -0,0 +1,227 @@
+package account
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/config"
+	"github.com/azure/azure-dev/cli/azd/pkg/tools/azcli"
+	"golang.org/x/exp/slices"
+)
+
+type Manager struct {
```

Comment:
Maybe add a code comment that we only inspect az values if az is installed and az isn't required for it to work?